dpdata/plugins/lammps.py

```diff
@@ -45,7 +62,18 @@ def to_system(self, data, file_name: FileType, frame_idx=0, **kwargs):
 class LAMMPSDumpFormat(Format):
     @Format.post("shift_orig_zero")
     def from_system(
-        self, file_name, type_map=None, begin=0, step=1, unwrap=False, **kwargs
+        self,
+        file_name,
+        type_map=None,
+        begin=0,
+        step=1,
+        unwrap=False,
+        input_name=None,
```

Comment:
Type hint and doc should be added for the new argument that is exposed to users.